buildtools/dtrace2sdt.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+FILE=$1
+TGT=$2
+
+if [[ -n "$TGT" ]]; then
+	exec 1>$TGT
+fi
+cat <<EOF
+#ifndef AUTO_PROBE_$$
+#define AUTO_PROBE_$$
+
+#include <sys/sdt.h>
+
+EOF
+
+cat $FILE | egrep '^#define.*\(' | \
+	sed -e 's/^#define[	 ]*//g; s/).*/)/g;' | \
+	grep -v _ENABLED | sort | uniq | \
+	awk '{
+		match($0, "^([^_]+)", appname);
+		match($0, "_([^\\(]+)", _func);
+                match($0, "\\((.+)\\)", dargs);
```

Comment:
Tiny whitespace nit-- here to the end of the `if` looks indented one too many spaces.